src/Configuration/Definition/BaseExtractorConfigDefinition.php

```diff
@@ -46,7 +46,6 @@ protected function getDbNode(): ArrayNodeDefinition
 
         // @formatter:off
```

Comment:
Chybí ještě `/** @noinspection NullPointerExceptionInspection */`

Reply:
IMO to neni potreba ho tam zbytecne davat. Ty chyby jsou vyignorovane v `phpstan.neon`